ecosystem/sep-0001.md

```diff
@@ -129,6 +129,7 @@ is_unlimited | boolean | The number of tokens is dilutable at the issuer's discr
 is_asset_anchored | boolean | `true` if token can be redeemed for underlying asset, otherwise `false`
 anchor_asset_type | string | Type of asset anchored. Can be `fiat`, `crypto`, `stock`, `bond`, `commodity`, `realestate`, or `other`.
 anchor_asset | string | If anchored token, code / symbol for asset that token is anchored to. E.g. USD, BTC, SBUX, Address of real-estate investment property.
+asset_attestation | url | URL to a page hosting attestation(s) or other proof(s) of reserves.
```

Comment:
Minor edits, in the field name I think we should lead with attestation so attestations group naturally. We can drop the plurals too I think, because multiple proofs are proof, and so the plural is unnecessary.
```suggestion
attestation_of_reserve | url | URL to attestation or other proof, evidence, or verification of reserves.
```